src/reactors/PriorityOrderReactor.sol

```diff
@@ -21,6 +21,8 @@ contract PriorityOrderReactor is BaseReactor {
     error InvalidDeadline();
     /// @notice thrown when an order's auctionStartBlock is in the future
     error OrderNotFillable();
+    /// @notice thrown when an order's nonce is invalid
+    error InvalidNonce();
```

Comment:
Maybe AlreadyFilled is better here? Though could also be canceled

Reply:
Yea I think AlreadyFilled is good